src/Api/ApiClient.php

```diff
@@ -312,4 +328,38 @@ private function decodeData(array $responseData, SignatureDataFormatter $signatu
 		return $responseData;
 	}
 
+	private function logRequest(string $url, array $queries, array $requestData = null, Response $response)
+	{
+		if ($this->logger === null) {
+			return;
+		}
+
+		$context = [
+			'type' => 'csobRequest',
+			'response-code' => $response->getResponseCode()->getValue(),
+			'response-data' => $response->getData(),
+		];
+
+		if (count($queries) !== 0) {
+			$context['queries'] = $queries;
+		}
+
+		if ($requestData !== null) {
+			$context['request-data'] = $requestData;
+		}
+
+		if (count($_GET) !== 0) {
```

Comment:
Moc nechápu, k čemu je logování $_GET a $_POST dobré. Nedával bych to tam. Pokud to někdo v nějakém projektu potřebuje logovat, ať si to doplní ve vlastní implementaci `LoggerInterface`.